packages/connector-tencent-email/docs/config-template.json

```diff
@@ -0,0 +1,70 @@
+{
+  "accessKeyId": "<access-key-id>",
+  "accessKeySecret": "<access-key-secret>",
+  "region": "ap-hongkong",
```

Comment:
Should also use a placeholder value here

Reply:
does it mean `ap-hongkong`? Currently, Tencent Cloud only supports this.

Reply:
Oh, is it? OK then...